config.go

```diff
@@ -546,6 +564,30 @@ func loadConfig() (*config, []string, error) {
 			return nil, nil, err
 		}
 
+		// Ensure pool fee is valid
+		if cfg.PoolFee < 0 || cfg.PoolFee > 1 {
```

Comment:
This is fine. I think it makes sense to allow pools to set the bounds of the pool fee (0% or 100%), it is up to the user to determine if the pool fee being asked for is reasonable and make a decision on that.